docs/analyzer-rules/AZFW0011.md

```diff
@@ -0,0 +1,32 @@
+# AZFW0011: Invalid binding type
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00011|
+| **Category** |[Usage]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when a function is binding to a non-iterable type for a  blob container path.
+
+## Rule description
+
+For a function that has a BlobInput attribute with a container path, the parameter must be of iterable type such as `IEnumerable<T>`.
```

Comment:
nit: When using the `BlobInputAttribute` with a container path, the target parameter must be of iterable type such as `IEnumerable<T>`